src/session/model/Session.ts

```diff
@@ -50,5 +53,7 @@ export class Session {
         if (expires <= dateNowMilliseconds) {
             throw new SessionExpiredError(expires, dateNowMilliseconds);
         }
+
+        loggerInstance().debug("Session Verified");
```

Comment:
Rubbish, it adds no value. Imagine seeing lots of `Session Verified` logs in one place? What value would that give? If session verification failed there would be much more meaningful error message printed in the logs.